CONTRIBUTING.md

```diff
@@ -0,0 +1,42 @@
+## First steps
+Thank you for getting involved! Before you start, please make sure to familiarize yourself with our README.md
+
+Please take note of our Code of Conduct https://github.com/edgelesssys/edgelessrt/blob/master/CODE_OF_CONDUCT.md and follow it when interacting with our project.
+
+If you want to support our development:
+*	Add a GitHub Star to the project
+*	Share our projects on Social Media
+*	Join out Discord at https://discord.com/invite/rH8QTH56JN
+
+Edgeless RT is licensed under the [MIT License](https://github.com/edgelesssys/edgelessrt/blob/master/LICENSE). When contributing, you also need to agree to our [Contributor License Agreement](https://cla-assistant.io/edgelesssys/edgelessrt).
+
+## Development philosophy
+
+As coding guidelines, we use openenclave's [Development Guide](https://github.com/openenclave/openenclave/blob/master/docs/Community/DevelopmentGuide.md).
+
+## Pull request process
+
+Submissions should remain focused in scope and avoid containing unrelated commits.
+For pull requests, we employ the following workflow:
+1.	Fork the repository to your own GitHub account
+2.	Create a branch locally with a descriptive name
+3.	Commit changes to the branch
+4.	Write your code according to our development philosophy
+5.	Push changes to your fork
+6.  Clean up your commit history
+7.	Open a PR in our repository and summarize the changes in the desciption
+
+## Reporting issues and bugs
+This project uses the GitHub issue tracker. Please take a look at the previous issues before submitting to avoid duplicates. If you have a quick question ask us on our discord.
```

Comment:
Since we substituted Gitter with Discord, should we explicitly mention a channel on the discord? #edgeless-support seems like the right place, but then the channel's description should also include ert.

Reply:
@flxflx